rpc/gnmi/gnmignoissh-dialout-grpctunnel.md

```diff
@@ -79,19 +86,19 @@ In this example there are a group of gNMI target devices that are configured to
 To implement the above, there are multiple options on both the device and collector side with how they integrate to the gRPC tunnel. Below will illustrate the variants on each side of the tunnel independently as they are interchangeable across the tunnel interface.
 
 ### Target-side gNMI tunnel client as a separate binary
-This option is available to a target implementer that wants to leverage the tunnel implementation as provided in Go without the need to directly integrate the code into an existing gNMI server which might be implemented in another language. The existing gNMI Server can be run without modification.  The tunnel client can be launched as a separate binary that establishes a tunnel to a remote collector and forwards all gNMI requests over tunnel sessions to the running server via a local TCP connection between the two binaries.
+This option is available to a target implementer that wants to leverage the tunnel implementation as provided in Go without the need to directly integrate the code into an existing gNMI server which might be implemented in another language. The existing gNMI Server can be run without modification.  The tunnel client can be launched as a separate binary that establishes a tunnel to a remote collector and forwards all gNMI requests over tunnel sessions to the running tunnel server via a local TCP connection between the two binaries.
 
 ![](img/grpctunnelclient_standalone.png)
 
-The grpctunnel repository has an example client application that can be configured to perform this task by connecting to a remote tunnel server and redirecting incoming sessions over a TCP connection dialed from the tunnel client to the local gNMI server.
+The grpctunnel repository has an [example tunnel client application](https://github.com/openconfig/gnmi/tree/master/cmd/gnmi_collector) that can be configured to perform this task by connecting to a remote tunnel server and redirecting incoming sessions over a TCP connection dialed from the tunnel client to the local gNMI server.
```

Comment:
This is a link to the gnmi_collector.  The link we want is the grpctunnel standalone binary.  We might even want to provide an example command line.

Reply:
I am using the link to the client lib for now. Will update once the example is ready.